### PR TITLE
[RELEASE-1.31.0] [SRVCOM-2744] Disable http2 in webhooks

### PR DESCRIPTION
**Pull request by skonto**
Fixes JIRA #SRVCOM-2744

## Proposed Changes

- Similar to #2345

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -781,7 +781,7 @@ spec:
                 containers:
                   - name: knative-operator
                     # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                    image: registry.ci.openshift.org/knative/openshift-serverless-v1.31.0:openshift-knative-operator
```
Comment (skonto) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:784]: I need to revert his one.

**Comment by ReToCode** [comment]
Also probably need to wait for https://github.com/openshift-knative/eventing/pull/396 to be in?

File: hack/lib/mesh_resources/authorization-policies/helm/eventing-e2e0.yaml [hunk 2]
```diff
@@ -542,3 +542,21 @@ spec:
         - operation:
             ports:
               - "8022"
+---
```
Comment (ReToCode) [inline hack/lib/mesh_resources/authorization-policies/helm/eventing-e2e0.yaml:545]: We don't need that, so you need to wait for 
https://github.com/openshift-knative/knative-istio-authz-chart/pull/11
to be merged, then re-run (or undo manually)

**Comment by ReToCode** [comment]
@pierDipi @matzew can you check if eventing + reconciler-test bumps are ok?

**Comment by ReToCode** [comment]
waiting for https://github.com/openshift-knative/serverless-operator/pull/2350, the re-running tests.

**Comment by matzew** [comment]
@ReToCode the rekt-test updates match the corresponding 1.10 midstream "vendor" updates on eventing.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -781,7 +781,7 @@ spec:
...
+                    image: registry.ci.openshift.org/knative/openshift-serverless-nightly:openshift-knative-operator
```
Comment (ReToCode) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:784]: :)

**Comment by skonto** [comment]
kitchensink stuff fail cc @pierDipi @ReToCode should we merge?

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

/hold for tests

**Comment by skonto** [comment]
@pierDipi I will re-run but could you take a look if anything is suspicious from the Eventing side.

**Comment by pierDipi** [comment]
@maschmid is looking into kitchensink tests here https://github.com/openshift-knative/serverless-operator/pull/2358, I don't think it's related to this change as [those jobs are constantly failing on the branch as well](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-knative-serverless-operator-release-1.31-4.13-e2e-kitchensink-ocp-413)

I think it's safe to merge

/lgtm

feel free unhold once you're ready

**Pull request merged**